components/bio-formats-tools/src/loci/formats/tools/XMLValidate.java

```diff
@@ -47,17 +48,66 @@
  */
 public class XMLValidate {
 
-  public static void process(String label, BufferedReader in)
+  public static boolean validate(String label, BufferedReader in)
```

Comment:
Since this is now a new method, can we invert the order of the arguments to mirror the low-level `XMLTools.validateXML(String, String)` method?